packages/prosemirror-lwdita/src/document.ts

```diff
@@ -301,6 +301,7 @@ export function unTravel(prosemirrorDocument: Record<string, any>): JDita {
   }
 
   // handle the attributes
+  delete attributes['parent'];
```

Comment:
What does this do? It doesn't seem to be tested for anywhere?

Reply:
> It doesn't seem to be tested for anywhere?

The cypress test should cover this one!!

Reply:
@marmoure The test is passing before this change! Therefore there is no test for this...

Reply:
https://github.com/evolvedbinary/prosemirror-lwdita/commit/aedb23f171d776fdd93b20c28352055353ee7ce8
there was a test, I not sure how this got approved!!

Reply:
@marmoure that may have been my fault - I didn't mean to add the `.skip` can you remove the skip and check it works please